src/labels/CustomLabels.labels

```diff
@@ -877,23 +877,23 @@ day:      &apos;day&apos;</value>
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>Activity logging error on the MassEmailVolunteers page</shortDescription>
-        <value>- Logging of Activities must be enabled for Volunteer Jobs and Volunteer Shifts.  Please see the Volunteers for Salesforce Installation and Configuration Guide.</value>
+        <value>You must enable the ability to log Activities for Volunteer Jobs and Shifts. For more information, see the &lt;a target=&apos;_blank&apos; href=&apos;https://powerofus.force.com/V4S_Config_Guide&apos;&gt;Set Up and Configure Volunteers for Salesforce Guide&lt;/a&gt;.</value>
```

Comment:
@sam-knox - I am not sure this is linking as we'd like either. I saw this after I hit approve. I just tossed the label on a different page because I wasn't sure off hand how to make this one happen.